### PR TITLE
getAllBabysitters route

### PR DESCRIPTION
**Pull request by liela9**
built getAllBabysitters route in folder "server/routes/app/parents"

### CODE REVIEW

File: server/package-lock.json [hunk 1]
```diff
@@ -9,20 +9,11 @@
       "version": "1.0.0",
       "license": "ISC",
       "dependencies": {
-        "@types/bcrypt": "^5.0.2",
```
Comment (ido123ziv) [inline server/package-lock.json:12]: all this deletions in this file will cause merge conflicts in the future, if you didn't change any package better to take this file from main 
`git checkout main server/package-lock.json` from your local branch + push should do the trick

File: server/utils/db/db.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ const pool = new Pool({
   user: process.env.DATABASE_USERNAME,
   database: process.env.DATABASE_NAME,
   password: process.env.DATABASE_PASSWORD,
-  port: process.env.DATABASE_PORT ? +process.env.DATABASE_PORT : 5432,
+  port: process.env.DATABASE_PORT ? +process.env.DATABASE_PORT : 5433,
```
Comment (ido123ziv) [inline server/utils/db/db.ts:8]: better to change the compose default to 5433 as well if we merge this 
@EliGandin  please update the .env template, I'll handle compose

**Review by ido123ziv — COMMENTED** [review]
lgtm 🍖

File: server/index.ts [hunk 3]
```diff
@@ -28,6 +29,8 @@ headerSetup(app);
 app.use("/signup", signupRouter);
 app.use("/login", loginRouter);
 app.use("/", router);
+app.use('/api', router);
+app.use("/app/parents", parentsRouter)
```
Comment (EliGandin) [inline server/index.ts:33]: please use "/api/parents" route

File: server/index.ts [hunk 4]
```diff
@@ -28,6 +29,8 @@ headerSetup(app);
 app.use("/signup", signupRouter);
 app.use("/login", loginRouter);
 app.use("/", router);
+app.use('/api', router);
```
Comment (EliGandin) [inline server/index.ts:32]: delete please so you can handle the next comment

**Review by EliGandin — CHANGES_REQUESTED** [review]

File: server/index.ts (hunk 4 above)
Comment (liela9) [reply server/index.ts:32]: Will do

File: server/index.ts (hunk 3 above)
Comment (liela9) [reply server/index.ts:33]: Ok, then change the file 'routes/app' to 'routes/api' also?

**Review by EliGandin — APPROVED** [review]

**Pull request merged**